hw/riscv/tk1.c

```diff
@@ -550,10 +550,8 @@ static void tk1_board_init(MachineState *machine)
     }
 
     if (machine->ram_size != mc->default_ram_size) {
-        char *sz = size_to_str(mc->default_ram_size);
-        error_report("Invalid RAM size, should be %s.", sz);
-        g_free(sz);
```

Comment:
`size_to_str` requires us to call `g_free` when we're done. 

https://github.com/tillitis/qemu/blob/dfad11a8a4c9685a25956de7db2f0639fb8be7ad/util/cutils.c#L970-L976